docs/middleware/BLSPublicKeyCompendium.md

```diff
@@ -0,0 +1,23 @@
+# BLSPublicKeyCompendium
+
+This contract is shared by all AVSs and serves a single place for operators to connect their execution layer address to a bn254 public key. When operators opt into AVSs, the AVS contracts can read from the BLSPublicKeyCompendium in order to get the BLS public key of the operator. This contract also prevents against [rogue key attacks](https://xn--2-umb.com/22/bls-signatures/#rogue-key-attack).
+
+## Flows
+
+There is only one flow for this contract, which is a call from an operator to register their execution layer address with thier bn254 public key.
```

Comment:
I find the wording "register their execution layer address with thier bn254 public key" to be confusing. recommend changing it to something like "register a bn254 public key as controlled by their execution layer address". As-is, it feels like you are saying that they already have the bn254 key and are somehow registering an execution layer address to that key.

Reply:
Fixed cf883904